bot/exts/evergreen/issues.py

```diff
@@ -13,10 +15,10 @@
     404: "Issue/pull request not located! Please enter a valid number!",
     403: "Rate limit has been hit! Please try again later!"
 }
-
 MAX_REQUESTS = 10
-
 REQUEST_HEADERS = dict()
+PYDIS_REPOS = "https://api.github.com/orgs/python-discord/repos"
```

Comment:
If we want to keep the cog general, we shouldn’t use `pydis` in any of the variable or function names. Additionally, the organisation name should be configurable.

Reply:
I kept the organisation name default since you had told that it would be restricted to python-discord repositories only.

Reply:
Yes, the default should be `python-discord`, but we still want to support overriding.